main.go

```diff
@@ -105,12 +118,22 @@ func downloadImages(url *url.URL, pageCount int, downloadFolder string) error {
 	return nil
 }
 
+func getBookTitle(url *url.URL) (string, error) {
+	configjs, err := downloadConfigJSFile(url)
```

Comment:
The config js file is already being downloaded before the ebook is being downloaded. Doing this in here, too, adds unnecessary overhead. It would be much better to just pass the configjs into a function that then extracts the book title if the flag for it is set